shm/Game.cpp

```diff
@@ -9,4 +10,31 @@ bool Game::checkWinCondition() const {
 
 bool Game::checkLoseCondition() const {
     return (player_->getMoney() == 0);
-}
\ No newline at end of file
+}
+
+void Game::PrintOptions() {
+    std::cout << "Ahoy captain! We're waiting for your commands! \n";
+    std::cout << "0. Exit \n";
+    std::cout << "1. Travel \n";
+    std::cout << "2. Buy \n";
+    std::cout << "3. Sell \n";
+}
```

Comment:
I think it's better to user to start from 1 and put exit as a last option

Reply:
I concur

Reply:
thx:)